doc/plots.rst

```diff
@@ -0,0 +1,116 @@
+#####
+Plots
+#####
+
+The **Plots** pane shows the static figures and images created during your session.
+It will show you plots from the :doc:`ipythonconsole`, produced by your code in the :doc:`editor` or generated by the :doc:`variableexplorer` allowing you to interact with them in several ways.
+
+.. image:: images/plots/plots-execution.gif
+   :align: center
+   :alt: Spyder main window showing execution for the Plots pane
+
+|
+
+The figures shown in the Plots pane are those associated with the currently active :guilabel:`Console` tab; if you switch consoles, the list of plots displayed (or none at all, if a new console) will change accordingly.
+
+.. image:: images/plots/plots-switch.gif
+   :align: center
+   :alt: Spyder's plots pane and console showing switching consoles
+
+|
+
+
+
+============
+Options Menu
+============
+
+The options menu in the top right of the :guilabel:`Plots` pane offers several ways to customize how your plots are displayed.
+
+
+~~~~~~~~~~~~~~~~~~~~
+Mute inline plotting
+~~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Mute inline plotting` option is enabled by default, preventing your plots from appearing in the Console.
+If you deactivate this option, figures will display in both the Plots pane and the Console.
+
+.. image:: images/plots/plots-inline-plotting.gif
```

Comment:
For me I think it's not so clear what's happening in this gif because the same figure is generated twice.

So I have a suggestion to improve it: the file in the editor should generate two figures in two separate cells. After running the first cell, you woild deactivate "Mute inline plotting" and run the second one. I think that way it'd easier for the reader to grasp what's happening after that option is deactivated.

Reply:
I thought about that but that just adds a lot of extra complexity we'd have to explain (e.g. code cells, etc) and I don't think any clarity benefit is worth the potential confusion, never mind having to redo it again when I thought we agreed what we wanted Jaunita to do.

Reply:
> I thought about that but that just adds a lot of extra complexity we'd have to explain (e.g. code cells, etc) and I don't think any clarity benefit is worth the potential confusion

Well, I think there's no need to explain what code cells are. Just that the file generates two plots and for that it'd need to have two cells.

> never mind having to redo it again when I thought we agreed what we wanted Jaunita to do.

Yeah, I just noticed this detail after seeing the gif for the second time. I really think it'd help to improve the meaning of this gif.

@juanis2112, what do you think?

Reply:
Ah okay; it doesn't have to be code cells, it can just be running lines through a selection. I did notice the same thing when I saw it, but I'm just not sure if its worth the time and complexity cost to change it at this point...I'm not blocking my approve on it at least.

Reply:
> it doesn't have to be code cells, it can just be running lines through a selection

Ok, that's true too.

> I'm not blocking my approve on it at least.

If @juanis2112 thinks it's fine as it is, then I'll approve and merge. That's why I asked her opinion above.

Reply:
So @juanis2112 told me she thinks having two images in this gif would be more confusing than having just one (i.e. as it is right now).

Since both of you agree on that, let's leave this gif without changes.

Reply:
:+1: